### ISSUE TITLE
HTML Code Completion

### ISSUE THREAD

**Issue by nirch** [plnkr/feedback#92]
HTML code completion is missing from the new editor.

**Comment by ggoodman** [comment]
@nirch are you referring to [emmet](https://docs.emmet.io/) support?

This is definitely something I would like to see return to the editor.

**Comment by nirch** [comment]
Yes, emmet support will be great. However, basic HTML tag code completion will be a good start :)
